CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
 
+## [v2.0.5] 2022-05-10
+
```

Comment:
I see we haven't updated headers under the Unreleased section for a while now. Mention here just in case. Other than that, thing looks good. ⭐